### ISSUE TITLE
Not able to run edda with my log file

### ISSUE THREAD

**Issue by shalusingh0411** [mongodb-labs/edda#124]
I have installed edda with "pip install edda" but i am not able to run it with my log file. I am using the log file of the same server, also the mongo instance on my local machine is on still i am getting this error.

shalu@shalu-desktop:~$ sudo edda /home/shalu/Desktop/1.log 
[sudo] password for shalu: 

Currently parsing log-file: /home/shalu/Desktop/1.log
[===================================================================================================]100%CRITICAL:edda.run_edda:No servers were found, exiting.

**Comment by tuanbuix** [comment]
Samantha I am experiencing the same issue w/ mongo version 3.0. python 2.7 package and pip installed . I am doing research for a company and they gave me a log file which is rather large so i will link it to you in your email for you to download as a dropbox. Thank you.

**Comment by timdonovanuk** [comment]
Is this tool abandoned now? Doesn't seem to work :/ 
'ImportError: No module named bson'

**Comment by tuanbuix** [comment]
i am not sure. it has comparability issues at the moment and still in developing.

Date: Fri, 5 Jun 2015 08:59:42 -0700

**Comment by samantharitter** [comment]
Hi there @timdonovanuk , it's possible that with the recent pymongo update this tool will also need to be updated.  For the time being, I would try this with an older version of pymongo and see if the import error goes away.  Additionally, the format of log messages may have changed in 3.0, this tool has not yet been updated for mongodb 3.0 logs.

@tuanbuix I thought we had mostly worked through your errors, are there other things you're having issues with at the moment?

Maintaining this project is something that I do when I have the time, but it's definitely in "maintenance" mode more than active feature development.

**Comment by tuanbuix** [comment]
Yeah we did!, we just need to match up the correct versions, was unable to parse the big log files. sample logs was a success. not sure in tim's case. he will probably needs to explain in greater detail. 

Date: Fri, 5 Jun 2015 09:57:12 -0700
